siriuspy/siriuspy/devices/bpm_fam.py

```diff
@@ -332,7 +332,8 @@ def config_mturn_acquisition(
             'adcswp': self._csbpm.AcqChan.ADCSwp,
             'adc': self._csbpm.AcqChan.ADC,
         }
-        acq_rate = dic.get(acq_rate.lower(), acq_rate)
+        if isinstance(acq_rate, str):
+            acq_rate = dic.get(acq_rate.lower())
```

Comment:
Hi @fernandohds564 , do you think we could check if the `int` values are valid here?

Reply:
Oh, sorry, you are right